README.rst

```diff
@@ -91,24 +90,18 @@ portable version you should observe the following install steps.*
       ez_setup.py for Setuptools
       <https://pypi.python.org/pypi/setuptools/#installation-instructions>`_.
 
-2. Setup a virtual environment with Virtual Python Environment builder
-   (optional)
-
-   You can setup a virtual environment to install OpenSlides as non-root
-   user. Make sure that you have installed Virtual Python Environment
-   builder on your system.
-
-   To install latest Virtual Python Environment builder, open command line (cmd)
-   and run::
+2. Setup a virtual Python environment using the virtual environment (venv)
+   package for Python (optional)
 
-       > easy_install https://pypi.python.org/packages/source/v/virtualenv/virtualenv-x.y.z.tar.gz
+   You can setup a virtual Python environment to install OpenSlides as
+   non-root user.
 
    Create your OpenSlides directory, change to it, setup and activate the
    virtual environment::
 
        > md OpenSlides
        > cd OpenSlides
-       > virtualenv .virtualenv
+       > python3 -m venv .virtualenv
```

Comment:
I tested it with Python 3.4 (32bit) under Windows 7. Please use the following lines in README:
```
python -m venv .virtualenv
.virtualenv\Scripts\activate.bat
```